### PR TITLE
Started the random module

### PR DESCRIPTION
**Pull request by martini97**
Started the random module with most of its functions

Signed-off-by: Alessandro Fernandes Martini alessandrofmartini@gmail.com

### CODE REVIEW

File: tests/modules/test_random.py [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+            foo = random.randint(0, 5)
+            print(a >= 0)
+            print(a <= 5)
+            """)
+
+    def test_random(self):
+        self.assertCodeExecution("""
+            import random
+            foo = random.random()
+            print(a >= 0)
```
Comment (wolfv) [inline tests/modules/test_random.py]: Shouldn't it be foo >=0 and foo < 1?

Comment (martini97) [inline tests/modules/test_random.py]: Yes, thank you

File: batavia/modules/random.js [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+    },
+
+    choice: function(list) {
+        var list = new batavia.types.List(list);
+        var rand_index = Math.floor(Math.random() * list.length);
+        return list[rand_index];
+    },
+
+    randint: function(lower, upper) {
+        return Math.ceil((Math.random() * (upper - lower)) + lower);
```
Comment (swenson) [inline batavia/modules/random.js]: I think we'll need to wrap this in a batavia.type.Int.

**Comment by phildini** [comment]
Hi there! We're closing this at this point due to inactivity, but feel free to re-open if you're interested in continuing it! Thanks!

**Pull request closed without merging**